### PR TITLE
Training data model

### PR DESCRIPTION
**Pull request by BharatGoru**
In this, we fine-tune our hyperparameters and found out the best possible parameters 

After that, we first used the decision tree classifier to train, test, and measure the accuracy of the model

### CODE REVIEW

**Comment by SeshaphaniGrandhe** [comment]
The training data model was successfully trained. you used  X_train, X _ test methods to split the data 80 percent of data will be split to train and 20 percent of data was split to test. By using the fit method the data will be trained. The Code runs perfect and great work.
